### DISCUSSION TITLE
Variable to automate link back to the previous note this new note is created from

### DISCUSSION THREAD

**Discussion by warm-july** [SilentVoid13/Templater#1088] [Help]
Works soft of like Extract note/heading function in the core plugin "Note Composer", there is a variable {{fromtTitle}} that enables linking to the previous note where this new note is linked from.

**Comment by AB1908** [comment]
Look at `tp.config.active_file` or something of the sort and use that to generate a wikilink. It's likely a TFile so use the `.basename` attribute or even the full path if that's necessary.

**Comment by gruff101** [comment]
Give the code below a try which creates a link back to the previous note that this new note was created from.

`let last_file = "";
let recent_leaf = this.app.workspace.getMostRecentLeaf(); 
let back_history = recent_leaf.history.backHistory; 
//skip when there is no back history 
//this can happen when the file gets created directly from the GUI 
if(back_history.length > 0) {
    // the last entry is the most recent file 
   last_file = "[[" + back_history[back_history.length - 1].title + "]]"; 
} `
